src/core/client/stream/tabs/Comments/ReplyList/__snapshots__/ReplyList.spec.tsx.snap

```diff
@@ -62,163 +62,3 @@ exports[`renders correctly 1`] = `
   />
 </ForwardRef(forwardRef)>
 `;
-
```

Comment:
Why are these snapshots removed? Looks like this part of the ReplyList tests has errors when run now.

Reply:
I honestly have no idea what is going on... I updated the tests, it worked, all was green, I pushed. Checking now.

Reply:
So weird, I ran all tests, all green, passed on my end. Then I removed initLocalState, ran all tests again, passed, all green. No changes to snapshots. So I then forced it with `npm run test ReplyList.spec` to tell it to just run the `ReplyList` tests and two snapshots popped out. This is super flaky and weird. Hopefully this fixes it now though.

Reply:
Okay, sorted, it's all good now.